src/styles/variables.scss

```diff
@@ -0,0 +1,10 @@
+//media queries
+$screen-tablet-max: 1280px;
+$screen-mobile-max: 768px;
+
+//fonts
+$ubuntu: "ubuntu";
```

Comment:
Don't see purpose of these font variable names. 
In case font family changes, these names will not make sense.

Reply:
Resolved sir